lib/addMethod/index.js

```diff
@@ -185,7 +190,9 @@ module.exports = function (methodName, config) {
         globalize.afterSuccess.call(threadneedle, config, result.body, params, result.response).done(resolve, reject);
       }, function (err) {
         logger.info(methodName+': running `afterFailure` hook', err);
-        globalize.afterFailure.call(threadneedle, config, err, params).done(reject, reject);
+        var payload = ( err.payload ? err.payload : err ),
```

Comment:
What's this bit for? Is this  to cover the `handleReject` on the socket hang up errors?